corehq/apps/reports/static/reports/js/tableau.js

```diff
@@ -8,9 +8,6 @@ hqDefine("reports/js/tableau", function () {
             method: 'post',
             url: initialPageData.reverse('get_tableau_server_ticket'),
             data: {
-                validate_hostname: initialPageData.get("validate_hostname"),
-                server_name: initialPageData.get("server_address"),
```

Comment:
Is there still any use including `server_address` in the initial page data? If not, it can be removed from being included in the page context. Same with `target_site`?